srf_generation/velocity_model_generation/q_model_generation/generate_qp_qs.py

```diff
@@ -168,13 +208,15 @@ def load_args():
     parser.add_argument(
         "--max_qs", type=float, default=MAX_QS, help="Maximum qs value to use."
     )
-
     parser.add_argument(
         "--useable_ram",
         type=float,
         help="Maximum available ram to use. In Gb",
         default=MEMORY,
     )
+    parser.add_argument(
+        "--basins", action="store_true", help="Vs based model in basins"
```

Comment:
I think we always want to store basins so may be better to have a no-basins flag and store it to the basins argument.